balance/src/balance.service.ts

```diff
@@ -10,40 +9,13 @@ export class BalanceService {
     @InjectModel(Balance.name) private balanceModel: Model<BalanceDocument>,
   ) { }
 
-  public async refillBalance({
-    balanceId,
-    transactionId,
-    refillSum,
-  }): Promise<ICurrentBalance | IError> {
-    const err = getStatusGrpcError(statusGrpc.NOT_FOUND);
-
-    // TODO кажется криво проверять id по длине для соответствия Id
-    // хотелось бы делать это в proto файле
-    if (balanceId.length !== 24) {
-      return err;
-    }
-    const balance = await this.balanceModel.findById(balanceId);
-    if (!balance) {
-      return err;
-    }
-
-    if (balance.transactions.find((t) => t.transactionId === transactionId)) {
-      return { total: balance.total };
-    }
-    balance.transactions.push({
-      transactionId: transactionId,
-      currentBalance: balance.total,
-      refillSum: refillSum,
-      transactionTime: new Date(),
+  public async refillBalance(balanceId, newBalance): Promise<IBalance> {
+    return await this.balanceModel.findByIdAndUpdate(balanceId, newBalance, {
+      new: true,
     });
-    balance.total = Number(balance.total) + Number(refillSum);
+  }
 
-    // TODO красиво избавиться от дублирования поиска транзакции
-    const newBalance = await this.balanceModel.findByIdAndUpdate(
-      balanceId,
-      balance,
-      { new: true },
-    );
-    return { total: newBalance.total };
+  public async getBalance(balanceId): Promise<IBalance> {
+    return await this.balanceModel.findById(balanceId);
```

Comment:
```suggestion
    return this.balanceModel.findById(balanceId);
```

await лишний в такой конструкции